framework/packages/abstract-sdk/src/apis/respond.rs

```diff
@@ -2,21 +2,21 @@ use crate::features::ModuleIdentification;
 use abstract_macros::with_abstract_event;
 use cosmwasm_std::{Attribute, Response};
 
-/// Wrap a [`Response`] with an abstract-specific event.
+/// Construct a [`Response`] with an abstract-specific event.
 /// The event contains the contract name and the action, and any additional attributes.
 pub trait AbstractResponse: ModuleIdentification {
     /// Respond with an abstract-specific event that contains the contract name and the action.
-    fn tag_response(&self, response: Response, action: impl Into<String>) -> Response {
-        self.custom_tag_response(response, action, Vec::<Attribute>::new())
+    fn tag_response(&self, action: impl Into<String>) -> Response {
```

Comment:
Should we rename this to just `response`?

and `custom_response` below

Reply:
Sure, updated! :wink: